python/cudf/tests/test_binops.py

```diff
@@ -98,6 +99,36 @@ def test_series_bitwise_binop(binop, obj_class, lhs_dtype, rhs_dtype):
     np.testing.assert_almost_equal(result.to_array(), binop(arr1, arr2))
 
 
+_logical_binops = [
+    (operator.and_, operator.and_),
+    (operator.or_, operator.or_),
+    (np.logical_and, cudf.logical_and),
+    (np.logical_or, cudf.logical_or),
+]
+
+
+@pytest.mark.parametrize('lhstype', _int_types + [np.bool_])
+@pytest.mark.parametrize('rhstype', _int_types + [np.bool_])
+@pytest.mark.parametrize('binop,cubinop', _logical_binops)
+def test_series_logical_binop(lhstype, rhstype, binop, cubinop):
+    import pandas as pd
+
+    arr1 = pd.Series(np.random.choice([True, False], 10))
+    if lhstype is not np.bool_:
+        arr1 = arr1 * (np.random.random(10) * 100).astype(lhstype)
+    sr1 = Series(arr1)
+
+    arr2 = pd.Series(np.random.choice([True, False], 10))
+    if rhstype is not np.bool_:
+        arr2 = arr2 * (np.random.random(10) * 100).astype(rhstype)
+    sr2 = Series(arr2)
+
+    result = cubinop(sr1, sr2)
+    expect = binop(arr1, arr2)
+
+    np.testing.assert_almost_equal(result.to_array(), expect)
```

Comment:
Are we able to assert Pandas behaviour more directly with `assert_eq(result, expect)`?